src/Common/tests/System/Diagnostics/RemoteExecutorTestBase.cs

```diff
@@ -132,6 +138,64 @@ public abstract class RemoteExecutorTestBase : FileCleanupTestBase
             return RemoteInvoke(method.GetMethodInfo(), new[] { unparsedArg }, options);
         }
 
+#if uap
+        /// <summary>Invokes the method from this assembly in another process using the specified arguments.</summary>
+        /// <param name="method">The method to invoke.</param>
+        /// <param name="args">The arguments to pass to the method.</param>
+        /// <param name="start">true if this function should Start the Process; false if that responsibility is left up to the caller.</param>
+        /// <param name="psi">The ProcessStartInfo to use, or null for a default.</param>
+        private static RemoteInvokeHandle RemoteInvoke(MethodInfo method, string[] args, RemoteInvokeOptions options)
+        {
+            options = options ?? new RemoteInvokeOptions();
+
+            // Verify the specified method is and that it returns an int (the exit code),
+            // and that if it accepts any arguments, they're all strings.
+            Assert.True(method.ReturnType == typeof(int) || method.ReturnType == typeof(Task<int>));
+            Assert.All(method.GetParameters(), pi => Assert.Equal(typeof(string), pi.ParameterType));
+
+            // And make sure it's in this assembly.  This isn't critical, but it helps with deployment to know
+            // that the method to invoke is available because we're already running in this assembly.
+            Type t = method.DeclaringType;
+            Assembly a = t.GetTypeInfo().Assembly;
+            Assert.Equal(typeof(RemoteExecutorTestBase).GetTypeInfo().Assembly, a);
+
+            using (var remoteExecutionService = new AppServiceConnection())
+            {
+                // Here, we use the app service name defined in the app service provider's Package.appxmanifest file in the <Extension> section.
+                remoteExecutionService.AppServiceName = "com.microsoft.corefxuaptests";
+                remoteExecutionService.PackageFamilyName = Package.Current.Id.FamilyName;
+
+                var status = remoteExecutionService.OpenAsync().GetAwaiter().GetResult();
```

Comment:
My understanding is the communication here is one way. I mean there is no callback to the caller thread. the docs didn't mention anything like that and I didn't notice any issue while running it so far.